float64 target_bearing
float64 current_bearing
float64 distance_from_target
uint8 command